app/assets/stylesheets/application.scss

```diff
@@ -1,5 +1,26 @@
-// /*
-// *= require_tree
-// *= require_self
-// *= require foundation_and_overrides
-// */
+//= require_tree
+//= require_self
+//= require foundation_and_overrides
+
+$body-color: #d6dbe0;
```

Comment:
This custom scss should live in its own file which is required in this application.scss file. This should be reserved for pulling in other files